source/includes/gcp/_backend_services.md

```diff
@@ -1 +1,15 @@
-#### Backend services
\ No newline at end of file
+#### Backend services
+
+<!-------------------- DELETE A BACKEND SERVICE -------------------->
+
+##### Delete a backend service
+
+```shell
+curl -X DELETE \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcpservicecode/gcpenvironment/backendservices/2570199154720991429"
```

Comment:
Can you change the URL to match the rest of the documentation:

`https://cloudmc_endpoint/v1/services/gcp/test-area/backendservices/2570199154720991429`

Reply:
Updated the url to keep it consistent with rest of the api doc.